packages/esm-patient-labs-app/src/config-schema.ts

```diff
@@ -41,16 +41,53 @@ export const configSchema = {
       _default: '52a447d3-a64a-11e3-9aeb-50e549534c5e',
     },
   },
+  labTests: {
+    _type: Type.Array,
+    _elements: {
+      labTestUuid: {
+        _type: Type.UUID,
+        _description: "UUID for the 'Lab' Test that require order reason",
+        _default: '',
+      },
+      labTestOrderReasons: {
+        _type: Type.Array,
+        _elements: {
+          uuid: {
+            _type: Type.UUID,
+            _description: 'Concept UUID representing the order reason',
+          },
+          label: {
+            _type: Type.String,
+            _default: null,
+            _description: 'The label for order reason',
```

Comment:
Defer to Ian's line of reasoning above, but if we're to keep this property, a more appropriate description could be:

```suggestion
            _description: 'The label for order reason concept answer',
```